### PR TITLE
Account stamt Estate UI changes

### PR DESCRIPTION
**Pull request by kshitijzutshi**
PLEASE KEEP THIS PR OPEN, WILL ADD MORE HERE

### CODE REVIEW

File: web/rainmaker/dev-packages/egov-estate-dev/src/ui-config/screens/specs/estate/searchResource/estateApplicationAccountStatementGen.js [hunk 1]
```diff
@@ -9,11 +9,11 @@ import {
     getDateField,
     getLabel
   } from "egov-ui-framework/ui-config/screens/specs/utils";
-  import { getSearchResults, getCount } from "../../../../..//ui-utils/commons";
+  import { getSearchResults, getCount } from "../../../../../ui-utils/commons";
   import { handleScreenConfigurationFieldChange as handleField, prepareFinalObject,toggleSnackbar } from "egov-ui-framework/ui-redux/screen-configuration/actions";
   import {
     searchApiCallAccountStatement
-  } from "./searchAccountStatementFunction";
+  } from "./searchAccountStatementFunction.js";
```
Comment (tejaswi-transerve) [inline web/rainmaker/dev-packages/egov-estate-dev/src/ui-config/screens/specs/estate/searchResource/estateApplicationAccountStatementGen.js:16]: .js ??

Comment (kshitijzutshi) [reply web/rainmaker/dev-packages/egov-estate-dev/src/ui-config/screens/specs/estate/searchResource/estateApplicationAccountStatementGen.js]: Changed n committed

**Comment by kshitijzutshi** [comment]
acc stmt API integrated. 
File Number used: FILE15
image

**Comment by kshitijzutshi** [comment]
account statement col. changes UI made, added commit

**Comment by kshitijzutshi** [comment]
@tejaswi-transerve File changes Acc stmt comments rem.

**Pull request merged**